openpype/plugins/publish/integrate.py

```diff
@@ -105,6 +105,7 @@ class IntegrateAsset(pyblish.api.InstancePlugin):
                 "review",
                 "rendersetup",
                 "rig",
+                "rig.fbx",
```

Comment:
```suggestion
```
This is not needed - you are only appending that family for the publish process but the `rig` family is still present, and thus it'd still be integrated.